src/collections/_documentation/performance-monitoring/performance/event-detail.md

```diff
@@ -88,6 +88,71 @@ addGlobalEventProcess(event => {
 });
 ```
 
+For browser JavaScript applications using the `Tracing` integration, the `beforeNavigate` option  can be used to better group navigation/pageload transactions together based on URL.
+
+```javascript
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/apm';
+
+Sentry.init({
+	// ...
+  integrations: [
+    new ApmIntegrations.Tracing({
+      beforeNavigate: (location) => {        
+        // You could use your UI's routing library to find the matching
+		// route template here. We don't have one right now, so do some basic
+		// parameter replacements.
+        return location.pathname
+        .replace(/\d+/g, '<digits>')
+        .replace(/[a-f0-9]{32}/g, '<hash>');
+      },
+    }),
+  ],
+});
+```
+
+**Adding Query Information and Parameters to Spans**
+
+Currently, every tag has a maximum character limit of 200 characters. Tags over the 200 character limit will become truncated, losing potentially important information. To retain this data, you can split data over several tags instead.
+
+For example, a 200+ character tagged database query:
+
+`SELECT first_column, second_column, third_columns, fourth_column, fifth_column, sixth_column FROM this_is_a_long_table_name WHERE first_column=some_value AND second_column=some_other_value AND third_column=yet_another_value`
+
+The 200+ character query above will become truncated to:
+
+`SELECT first_column, second_column, third_columns, fourth_column, fifth_column, sixth_column FROM this_is_a_long_table_name WHERE first_column=some_value AND second_column=some_other_value AND third_c`
+
+Instead, using `span.set_tag` you could split the details of this query over several tags. This could be done over `columns`, `tables`, `conditions`, in this case, resulting in three different tags:
+
+```python
```

Comment:
@MimiDumpling do you know if it's a good idea if we should provide examples for each language/framework that has Performance support? For example, Node.js, Python, and browser JavaScript.

Reply:
Actually looking at the other examples should we do everything in JavaScript first, and add more languages over time? (Or now I guess)

I could flip the example query to a long url, and split by URL params instead so that its more applicable?

Reply:
Ideally, we'd have an example for every supported lang, like Alberto suggested. If you can get me the content, I can add a dropdown for each language.